plugins/services/src/js/containers/services/ServicesTable.js

```diff
@@ -109,7 +114,7 @@ class ServicesTable extends React.Component {
     return (
       <Link
         className="table-cell-link-primary text-overflow"
-        to={`/services/overview/${id}`}>
+        to={urlPrefix + id}>
```

Comment:
I believe storing in a variable will help debug. So adding right below `const urlPrefix` something like `const serviceLinkId = urlPrefix + id`

then in the return

```
<Link
        className="table-cell-link-primary text-overflow"
        to={serviceLinkId}>
        {service.getName()}
      </Link>
```